addons/mrp/models/mrp_abstract_workorder.py

```diff
@@ -2,6 +2,8 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from collections import defaultdict
+from re import findall as regex_findall
```

Comment:
abstract share the code between workorder and production wizard, as this tak is related only to the production wizard, move your code there :)

Reply:
But in the task, oma mentioned we want the same thing for workorder
https://www.odoo.com/web#action=333&active_id=809&cids=1&id=2223443&menu_id=4720&model=project.task&view_type=form
We still only do it for production wizard?

Reply:
i asked confirmation to oriane and it's not planned at the moment:)